lib/auth/auth_with_roles.go

```diff
@@ -5792,6 +5792,114 @@ func (a *ServerWithRoles) DeleteAllUserGroups(ctx context.Context) error {
 	return a.authServer.DeleteAllUserGroups(ctx)
 }
 
+// ListOktaImportRules returns a paginated list of all Okta import rule resources.
+func (a *ServerWithRoles) ListOktaImportRules(ctx context.Context, pageSize int, pageToken string) ([]types.OktaImportRule, string, error) {
+	if err := a.action(apidefaults.Namespace, types.KindOktaImportRule, types.VerbList); err != nil {
```

Comment:
I think we enforce the read perm when listing as well
```suggestion
	if err := a.action(apidefaults.Namespace, types.KindOktaImportRule, types.VerbList, types.VerbRead); err != nil {
```